src/devices/r900.c

```diff
@@ -0,0 +1,297 @@
+/** @file
+    Neptune r900 protocol decoder for r900 based flow meters, tested with my water meter.
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+ */
+
+/**
```

Comment:
this need to be `/** @fn int r900_decode(r_device *decoder, bitbuffer_t *bitbuffer)`